javascript/packages/orchestrator/src/providers/k8s/kubeClient.ts

```diff
@@ -141,8 +141,8 @@ export class KubeClient extends Client {
 
     await this.createResource(podDef, true);
     if (podDef.metadata.labels["zombie-role"] !== ZombieRole.Temp) {
-      const serviceDef = genServiceDef(podDef);
-      await this.createResource(serviceDef, true);
+      const genFn = podDef.spec.delay ? genChaosDef : genServiceDef;
```

Comment:
Is not one or the other, we need to deploy both manifests if the `delay` is set. One to create the service, this allow us to refer to the pod by `dns` and the `chaos` to inject network failures.